nibabel/spatialimages.py

```diff
@@ -516,11 +516,11 @@ def __str__(self):
         shape = self.shape
         affine = self.affine
         return '\n'.join((str(self.__class__),
-                          'data shape %s' % (shape,),
+                          f'data shape {shape}',
                           'affine: ',
-                          '%s' % affine,
+                          f'{affine}',
```

Comment:
No advantage to f-strings here.

```suggestion
                          str(affine),
```

Unless you want to do something like combining the two lines: `'affine:\n{affine}'`